linter/whitelists/default.txt

```diff
@@ -20,4 +21,119 @@ TinyPNG
 trilobit
 Dalley
 PhotoSwipe
-Slack
\ No newline at end of file
+Slack
+Catalog
+Manager
+Backend
+Bing
+Bundle
+Bundles
+Changelanguage
+ChangeLanguage
+Twig
+Tweets
+Twitter
+Template
+Templates
+Templating
+Themes
+Theme
+Usability
+WhatsApp
+Widget
+Widgets
+Wizard
+backend
+contao
+estao
+geocoding
+Frontend
+Framework
+Footer
+theme
+symfony
```

Comment:
`Symfony` is a name, always uppercase.

Reply:
Corrected, but might be used in a bundle name as well